lib/msf/core/exploit/remote/http/moodle/login.rb

```diff
@@ -23,7 +23,7 @@ def moodle_login(user, pass, timeout = 20)
       'keep_cookies' => true
     }, timeout, 20) # typical redirect is 3-5, but it may do more if caching gets messed up on server
 
-    if !res || (res.code != 200) || !res.body.include?('<title>Dashboard</title>')
+    if !res || (res.code != 200) || !res.body.include?('<title>Dashboard')
```

Comment:
```suggestion
    # eg. `<title>Dashboard | moodle</title>` or `<title>Dashboard | New Site</title>`
    if !res || (res.code != 200) || !res.body.include?('<title>Dashboard [|] ')
```

Reply:
This change does not match the existing regex.